blue_bringup/scripts/gripper_calibration_service.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+
+"""This node provides a service to calibrate the grippers."""
+
+import rospy
+import numpy as np
+import actionlib
+from blue_msgs.srv import GripperPositionCalibration
+from controller_manager_msgs.srv import SwitchController
+from controller_manager_msgs.srv import LoadController
+from controller_manager_msgs.srv import ListControllers
+from std_srvs.srv import Trigger, TriggerRequest, TriggerResponse
+from blue_msgs.msg import MotorState
+from std_msgs.msg import Float64MultiArray
+from control_msgs.msg import (
+    GripperCommandAction,
+    GripperCommandGoal,
+)
+
+def get_gripper_state(motor_names):
```

Comment:
since we just pass the global `motor_names` variable in for the `motor_names` argument anyways, it seems like we can remove the argument here?

Reply:
also, maybe "get_gripper_actuator_position" or something would be more precise; state is pretty vague